code/modules/mob/living/simple_animal/bot/honkbot.dm

```diff
@@ -0,0 +1,400 @@
+/mob/living/simple_animal/bot/honkbot
+	name = "\improper Honkbot"
+	desc = "A little robot. It looks happy with its bike horn."
+	icon = 'icons/mob/aibots.dmi'
+	icon_state = "honkbot0"
+	density = FALSE
+	anchored = FALSE
+	health = 25
+	maxHealth = 25
+	damage_coeff = list(BRUTE = 0.9, BURN = 1.0, TOX = 0, CLONE = 0, STAMINA = 0, OXY = 0)
+	pass_flags = PASSMOB
+
+	radio_key = /obj/item/device/encryptionkey/headset_service //doesn't have security key
+	radio_channel = "Service" //Service
+	bot_type = HONK_BOT
+	model = "Honkbot"
+	bot_core_type = /obj/machinery/bot_core/honkbot
+	window_id = "autohonk"
+	window_name = "Honkomatic Bike Horn Unit v1.0.2"
+	//allow_pai = 1 //Damn right we'll pAI these
+	data_hud_type = DATA_HUD_SECURITY_ADVANCED // show jobs
+
+	var/honksound = 'sound/items/bikehorn.ogg' //customizable sound
+	var/goldenbikehorn = 0 //placeholder
+	var/spam_flag = 0
+	var/cooldowntime = 40
+	var/cooldowntimehorn = 10
+	var/cooldowntimeEmag = 5
+	var/mob/living/carbon/target
+	var/oldtarget_name
+	var/target_lastloc = FALSE	//Loc of target when arrested.
+	var/last_found = FALSE	//There's a delay
+	var/threatlevel = FALSE
+	var/declare_arrests = 0 // speak, you shall not, unless to Honk
+	var/idcheck = 1 //Chases unknowns
+	var/fcheck = 1 //And armed people
+	var/check_records = 1 //Doesn't care about criminals
+	var/arrest_type = 0
+	var/weaponscheck = 1
+
+/mob/living/simple_animal/bot/honkbot/Initialize()
+	. = ..()
+	icon_state = "honkbot[on]"
+	auto_patrol = 1
+	spawn(3)
+		var/datum/job/clown/J = new/datum/job/clown
+		access_card.access += J.get_access()
+		prev_access = access_card.access
+
+
+/mob/living/simple_animal/bot/honkbot/turn_on()
+	..()
+	icon_state = "honkbot[on]"
+
+/mob/living/simple_animal/bot/honkbot/turn_off()
+	..()
+	icon_state = "honkbot[on]"
+
+/mob/living/simple_animal/bot/honkbot/bot_reset()
+	..()
+	target = null
+	oldtarget_name = null
+	anchored = 0
+	walk_to(src,0)
+	last_found = world.time
+
+/mob/living/simple_animal/bot/honkbot/set_custom_texts()
+
+	text_hack = "You overload [name]'s sound control system"
```

Comment:
Oh come on, this is the FUNNY BOT! Give it some better texts! Maybe have it laugh evilly or something?

Reply:
Oh yes, an evil clown laugh would be appropriate.